src/app/main.py

```diff
@@ -31,13 +33,17 @@ def api() -> FastAPI:
     _api.add_middleware(RawContextMiddleware)
 
     _api.add_event_handler("startup", connect_to_elastic)
+    _api.add_event_handler("startup", create_start_app_handler(_api))
```

Comment:
:+1: 

alternatively you could use `@app.on_event("startup")`

Reply:
I liked how - this way - everything is done at one point, not "randomly" somewhere in the code. I therefore prefer this way, at least right now.

Reply:
fair point :)